src/sentry/static/sentry/app/views/releases/detail/overview/releaseStats.tsx

```diff
@@ -239,14 +273,31 @@ const Container = styled('div')`
   margin-bottom: ${space(3)};
 `;
 
-const LinkedStatsWrapper = styled('div')`
+const SectionContent = styled('div')``;
```

Comment:
shouldn't we use only the div?

Reply:
I had the plain `div` before but making it named component greatly improved code readability.